pkg/ceph/rgw/daemon.go

```diff
@@ -65,6 +67,13 @@ func generateConfigFiles(context *clusterd.Context, config *Config) error {
 		logger.Warningf("failed to create data directory %s: %+v", dataDir, err)
 	}
 
+	sslSuffix := ""
+	if config.Certificate != "" {
+		// the suffix is intended to be appended to the end of the rgw_frontends arg, immediately after the port.
+		// with ssl enabled, the port number must end with the letter s.
+		sslSuffix = fmt.Sprintf("s ssl_certificate=%s", config.Certificate)
```

Comment:
i'm not sure if i traced this correctly, but is this `config.Certificate` field the binary data that represents the cert? If so, do we need to do any escaping or sanitization?

Reply:
`config.Certificate` is actually a file path that has been loaded into the pod by mounting the secret as a file. I'll add a comment